wagtail_storages/tests/settings.py

```diff
@@ -46,12 +44,11 @@
 
 STATIC_ROOT = os.path.join(TESTS_PATH, "static")
 
-if WAGTAIL_VERSION >= (5, 2):
-    # when running tests with Wagtail 5.2+, we need to set the STATIC_URL explicitly,
-    # otherwise the test server will not start.
-    # This happens when running tests against the main branch of Wagtail
-    # so I am assuming that an upcoming release will require this.
-    STATIC_URL = "/static/"
+# when running tests with Wagtail 5.2+, we need to set the STATIC_URL explicitly,
+# otherwise the test server will not start.
+# This happens when running tests against the main branch of Wagtail
+# so I am assuming that an upcoming release will require this.
```

Comment:
**chore:** This comment can be removed now that the configuration's no longer conditional

Reply:
Removed the comment above